aries-site/src/examples/templates/dashboards/content/chartCards/CostByService.js

```diff
@@ -0,0 +1,169 @@
+      const nextValues = [];
+      consumptionData.forEach(datum => {
+        const dataPointDate = new Date(datum.endDate);
+        if (
+          dataPointDate > reportWindow.begin &&
+          dataPointDate < reportWindow.end
+        ) {
+          nextValues.push({
+            label: `${datum.service[0].toLocaleUpperCase()}${datum.service
+              .slice(1)
+              .toLocaleLowerCase()}`,
+            value: parseFloat(datum.cost.replace(/[$,]/gm, '')),
+            displayValue: datum.cost,
+            color: undefined,
+          });
+        }
+      });
+      nextValues
+        .sort((a, b) => b.value - a.value)
+        .forEach((value, index) => {
+          // eslint-disable-next-line no-param-reassign
+          value.color = chartColors[index % chartColors.length];
+        });
+      setValues(nextValues);
+    }
+  }, [chartColors, consumptionData, reportWindow.begin, reportWindow.end]);
+
+  // Calculate total cost
+  useEffect(() => {
+    let nextTotalCost = 0;
+    if (values) {
+      Object.keys(values).forEach(key => {
+        nextTotalCost += values[key].value;
+      });
+      setTotalCost(nextTotalCost);
+    }
+  }, [values]);
+
+  const grid = {
+    columns: ['auto', 'auto'],
+    rows: ['auto', 'auto'],
+    areas: {
+      xsmall: [
+        ['chart', 'measure'],
+        ['chart', 'legend'],
+      ],
+      small: [
+        ['chart', 'measure'],
+        ['chart', 'legend'],
+      ],
+      medium: [
+        ['chart', 'measure'],
+        ['chart', 'legend'],
+      ],
+      large: [
+        ['chart', 'measure'],
+        ['legend', 'legend'],
+      ],
+      xlarge: [
+        ['chart', 'measure'],
+        ['chart', 'legend'],
+      ],
+    },
+    gap: {
+      xsmall: 'medium',
+      small: 'medium',
+      medium: 'small',
+      large: 'medium',
+      xlarge: 'medium',
+    },
+  };
+
+  return (
+    <ChartCard title="Cost by Service" subtitle={period}>
+      {values && (
+        <Grid
+          columns={grid.columns}
+          rows={grid.rows}
+          areas={grid.areas[size]}
+          gap={grid.gap[size]}
+        >
+          <Box
+            gridArea="chart"
+            alignSelf="start"
+            height={{
+              max: `${parseMetricToNum(theme.global.size.small) +
+                parseMetricToNum(theme.global.size.xsmall)}px`,
+            }}
+          >
+            <Meter
+              values={values.map(value => ({
+                label: value.label,
+                value: value.value,
+              }))}
+              type="pie"
+              size="full"
+            />
+          </Box>
+          <Measure
+            gridArea="measure"
+            alignSelf="center"
+            name={{ label: { label: 'Total Cost', size: 'medium' } }}
+            value={{
+              value: Intl.NumberFormat('en-US', {
+                style: 'currency',
+                currency: 'USD',
+              }).format(totalCost),
```

Comment:
Looking for where the "Total Cost" and value get rendered but designs are calling for a '3px' gap

Reply:
Interesting. This is just being implemented with NameValuePair (same as the designs). Looks like we set the 3px [here in the theme](https://github.com/grommet/grommet/blob/db5be926eb7c2f791534f02dd55b0f9997e959db/src/js/themes/base.js#L1072), but it is not coming through. Any ideas?

Reply:
Seems like in https://design-system.hpe.design/components/namevaluelist?q=nam#vertical, the `3px` gets applied as a bottom margin to the name (which I believe we decided to do to avoid adding an extra div in the DOM). We stuck with gap for the concept but it gets translated to margin.


<img width="232" alt="Screen Shot 2022-02-11 at 10 51 59 AM" src="https://user-images.githubusercontent.com/12522275/153652149-a0bf1602-f3b2-4a3d-b7a6-9011be8f5104.png">

I think if we aren't passing the name as a string to NameValueList, it is assuming it's a custom element and not applying the margin. We might just need to update `Measure` to include the `3px` if that's the case.

Reply:
Hmmm... interesting. The lack of gap when custom node vs. string is surprising to me. The explanation makes sense, but unexpected behavior as a caller.

I will make the change in Measure.

Reply:
Done.

Reply:
hmm maybe we can look back at our grommet implementation I think we just wanted to provide flexibility